src/cdomain/value/cdomains/int/enumsDomain.ml

```diff
@@ -362,6 +375,13 @@ module Enums : S with type int_t = Z.t = struct
     | Inc e, Some (c, m) -> Inc (BISet.filter (contains c m) e)
     | _ -> a
 
+  let refine_with_bitfield ik x (z,o) =
+    match BitfieldDomain.Bitfield.to_int (z,o) with 
+    | Some y ->     
+      meet ik x (Inc (BISet.singleton y))
+    | _ ->
+      x
+
```

Comment:
In line with #1675 (and #1673) this should now be modified to ensure no new elements appear.

```suggestion
  let refine_with_bitfield ik x (z,o) =
    match x, BitfieldDomain.Bitfield.to_int (z,o) with 
    | Inc, Some y ->     
      meet ik x (Inc (BISet.singleton y))
    | _ ->
      x

```